### PR TITLE
Update AMQ Streams to 1.8.0 + migration to UBI 8

### PR DESCRIPTION
**Pull request by kyguy**
- Updates version numbers, names, and dist-git branches to AMQ Streams 1.8.0
- Updates base images to use UBI8

Will save the install/example files, MD5 , and CSV file changes for later PRs

### CODE REVIEW

File: bridge/modules/bridge/module.yaml [hunk 1]
```diff
@@ -19,12 +19,13 @@ packages:
...
   content_sets:
     x86_64:
-      - rhel-7-server-rpms
+      - rhel-8-for-x86_64-baseos-rpms
   install:
     - unzip
     - tar
     - rsync
     - shadow-utils # groupadd
+    - gzip
```
Comment (ppatierno) [inline bridge/modules/bridge/module.yaml:28]: why this addition?

File: kafka/modules/kafka/2.8.0/module.yaml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 envs:
   - name: "COM_REDHAT_COMPONENT"
-    value: "amqstreams-kafka-26-container"
+    value: "amqstreams-kafka-28-container"
 
 artifacts:
-  - md5: 8f3c4cb06479d333f10a5666c5968d55
-    name: streams-ocp-26.zip
+  - md5: c281ec8d1ed15f9ca99d6a1606371ad2
+    name: streams-ocp-27.zip
```
Comment (ppatierno) [inline kafka/modules/kafka/2.8.0/module.yaml:12]: streams-ocp-28 maybe?

File: bridge/modules/bridge/module.yaml (hunk 1 above)
Comment (kyguy) [reply bridge/modules/bridge/module.yaml:28]: `gzip` is needed by tar to handle `tar.gz` files. It was shipped automatically with `ubi7-minimal` image but not with the new `ubi8-minimal` image so we have to install it explicitly

Comment (ppatierno) [reply bridge/modules/bridge/module.yaml:28]: Why did they remove it!?? Anyway, thanks!

**Review by ppatierno — APPROVED** [review]

**Pull request merged**